crates/cast/tests/cli/main.rs

```diff
@@ -1411,6 +1411,71 @@ casttest!(block_number_hash, |_prj, cmd| {
     assert_eq!(s.trim().parse::<u64>().unwrap(), 1, "{s}")
 });
 
+// ... existing code ...
+
+casttest!(send_custom_error, async |prj, cmd| {
+    // Start anvil
+    let (_api, handle) = anvil::spawn(NodeConfig::test()).await;
+    let endpoint = handle.http_endpoint();
+
+    prj.add_source(
+        "SimpleStorage",
+        r#"
+    contract SimpleStorage {
+
+    error ValueTooHigh(uint256 providedValue, uint256 maxValue);
+
+    function setValueTo101() public {
+        revert ValueTooHigh({ providedValue: 101, maxValue: 100 });
+    }
+}
+    "#,
+    )
+    .unwrap();
+
+    // Deploy the contract
+    let output = cmd
+        .forge_fuse()
+        .args([
+            "create",
+            "./src/SimpleStorage.sol:SimpleStorage",
+            "--rpc-url",
+            &endpoint,
+            "--private-key",
+            "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
```

Comment:
```suggestion
        .args([
            "create",
            "./src/SimpleStorage.sol:SimpleStorage",
            "--rpc-url",
            &endpoint,
            "--broadcast",
            "--private-key",
            "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
```

we've recently added a requirement for passing the `--broadcast` flag to `forge create`, hence the failing test

Reply:
Solved, thanks